sdk/pinocchio/src/instruction.rs

```diff
@@ -171,6 +179,11 @@ impl<'a> From<&'a AccountInfo> for AccountMeta<'a> {
 #[derive(Debug, Clone)]
 pub struct Seed<'a> {
     /// Seed bytes.
+    /// JC nit: same here, since a slice is a pointer followed by a length, maybe
+    /// we can do the same thing here and avoid the phantom data?
+    /// On the flip side, that would make the types work inconsistently on
+    /// non-Solana architectures, ie. a 32-bit build would not be the as a 64-bit
+    /// build. But since Solana is always 64-bit, maybe we're ok?
```

Comment:
I will double check, I had problems using slices before and it seems other people as well: https://github.com/cavemanloverboy/solana-nostd-entrypoint/issues/11